transport/TransportImpl.h

```diff
@@ -41,6 +41,9 @@ struct SctpConfig;
 
 namespace transport
 {
+
+typedef memory::RandomAccessBacklog<memory::UniquePacket, 512> pacing_queue_t;
```

Comment:
I like this. but I guess there is a kind of a rule in this project to not create alias for types like this. This is because it creates an indirection for who is reading the code I guess. But the team has changed so maybe we can discuss with @olofkallander if we can change some code rules . My vote is yes on this one :)

Reply:
Let's keep it this way then. When Olof is back, we'll talk.